packages/sdks/output/qwik/src/blocks/submit-button/submit-button.tsx

```diff
@@ -0,0 +1,89 @@
+// GENERATED BY MITOSIS
+
+import { component$, useStore, h, Fragment } from "@builder.io/qwik";
+export const COMPONENT = {
+  "@type": "@builder.io/mitosis/component",
+  imports: [],
+  exports: {},
+  inputs: [],
+  meta: {},
+  refs: {},
+  state: {},
+  children: [
+    {
+      "@type": "@builder.io/mitosis/node",
+      name: "button",
+      meta: {},
+      scope: {},
+      properties: {
+        type: "submit",
+      },
+      bindings: {
+        _spread: {
+          code: "props.attributes",
+        },
+      },
+      children: [
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {
+            _text: "\n      ",
+          },
+          bindings: {},
+          children: [],
+        },
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {},
+          bindings: {
+            _text: {
+              code: "props.text",
+            },
+          },
+          children: [],
+        },
+        {
+          "@type": "@builder.io/mitosis/node",
+          name: "div",
+          meta: {},
+          scope: {},
+          properties: {
+            _text: "\n    ",
+          },
+          bindings: {},
+          children: [],
+        },
+      ],
+    },
+  ],
+  hooks: {},
+  context: {
+    get: {},
+    set: {},
+  },
+  name: "SubmitButton",
+  subComponents: [],
+  interfaces: [
+    "export interface ButtonProps {\n  attributes?: any;\n  text?: string;\n}",
+  ],
+  propsTypeRef: "ButtonProps",
+};
+export interface ButtonProps {
+  attributes?: any;
+  text?: string;
+}
+const SubmitButton = component$((props: ButtonProps) => {
+  const state = useStore({});
```

Comment:
nit pick: we don't need to generate state if no state is used. iirc in mitosis there is a helper function to detect if there actually is any state used (you can peek at how the other generators detect this)